src/org.ts

```diff
@@ -193,6 +193,25 @@ export class Org extends AsyncCreatable<Org.Options> {
     });
   }
 
+  /**
+   *
+   * @param sandboxReq SandboxRequest options to create the sandbox with
+   * @param sandboxName
+   * @param options Wait: The amount of time to wait before timing out, Interval: The time interval between polling
+   */
+  public async cloneSandbox(
+    sandboxReq: SandboxRequest,
+    sandboxName: string,
+    options: { wait?: Duration; interval?: Duration }
+  ): Promise<SandboxProcessObject> {
+    if (sandboxName) {
+      sandboxReq.SourceId = await this.querySandboxInfoIdBySandboxName(sandboxName);
+      this.logger.debug('Clone sandbox sourceId %s', sandboxReq.SourceId);
+    }
```

Comment:
Is it still a clone if the `sandboxName` was not provided?  I.e., should we throw an error if it's not provided?

Reply:
Not sure, originally it was written like that in toolbelt, which means if clone can't find a matching sandbox to clone from it would create one for you nonetheless. by throwing instead we might break compatibility, though I've been told this are low traffic commands if you know what I mean, so might be a good idea to throw when we cannot find a parent sandbox to clone from ? what you think @mshanemc ?

Reply:
If `sandboxName` query returns less than 1 result (`querySandboxInfoIdBySandboxName `) it will throw.
No sandbox will be created if there are no matches